xcodeproj/internal/runner.template.sh

```diff
@@ -74,10 +74,10 @@ unset BAZELISK_SKIP_WRAPPER
 
 readonly bazel_cmd=(
   env -i
-  HOME="$HOME"
+  HOME="${HOME:-}"
   PATH="/opt/homebrew/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
-  TERM="$TERM"
-  USER="$USER"
+  TERM="${TERM:-}"
+  USER="${USER:-}"
```

Comment:
Instead let's create an `env_values` array with the always set ones, then check for values and add them to the array, and finally expand the array.